File: Questions in Java/1040.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {

        Scanner input = new Scanner(System.in); //--Entrada de Dados

        // Variavel :)

        Float Notas1 = input.nextFloat();
        Float Notas2 = input.nextFloat();
        Float Notas3 = input.nextFloat();
        Float Notas4 = input.nextFloat();

        // Problema :)

        float Media =  (Notas1*2 + Notas2*3 + Notas3*4 + Notas4*1)/10;

        System.out.println("Media: "+ String.format("%.1f",Media));

        if(Media < 5){
            System.out.println("Aluno reprovado.");
        }

        else if(Media >= 7){
            System.out.println("Aluno aprovado.");
        }

        else{

            System.out.println("Aluno em exame.");

            //variaveis finais

            Float Notas5 = input.nextFloat();
            input.close();

            // Problema finais

            System.out.println("Nota do exame: " + String.format("%.1f",Notas5));

            Float media2 = (Media+Notas5)/2;

            if(media2 > 5 ){
                System.out.println("Aluno aprovado.");
            }
            if(media2 < 5 ){
                System.out.println("Aluno reprovado.");
            }

            System.out.println("Media final: " + String.format("%.1f",media2));

        }
    }
 
}

File: Question-in-Python/1040.py
Nota1,Nota2,Nota3,Nota4 = map(float,input().split(' '))#--Entrada de dados

Media = ((Nota1*2)+(Nota2*3)+(Nota3*4)+(Nota4*1))/10

print('Media: %0.1f' % Media )

if (Media >= 7):
    print('Aluno aprovado.' )

elif(Media<5):
    print('Aluno reprovado.')

if (5.0<=Media<=6.9):
    print('Aluno em exame.')

    Nota5 = float(input())
    print('Nota do exame: %0.1f' % Nota5)
    Media2 = (Media+Nota5)/2

    if (Media2 >= 5.0):
            print('Aluno aprovado.')
    else:
        print('Aluno reprovado.')

    print('Media final: %0.1f' % Media2)